### PR TITLE
Improved grammar

### PR DESCRIPTION
**Pull request by ajacocks**
Slightly improved grammar on RHEL workshop

### CODE REVIEW

**Comment by ajacocks** [comment]
Updated Ansible workshop to use new wildcard ssl certificates

**Review by benblasco — COMMENTED** [review]
No input on Ansible workshop content, but I made a minor grammar tweak to the RHEL 8 instruction.

File: content/workshops/ansible_automation/exercise2.0.adoc [hunk 1]
```diff
@@ -63,7 +63,7 @@ vim inventory
 
 === Step 6: Identify variables
 
-Fill out a couple variables in the inventory file, *admin_password* and *pg_password*:
+Fill out a couple variables in the inventory file, *admin_password* and *pg_password*. Also, fill in the *web_server_ssl_cert* and *web_server_ssl_key* fields, and uncomment both, as shown:
```
Comment (Tekwriter1221) [inline content/workshops/ansible_automation/exercise2.0.adoc:66]: Fill out a couple of variables in the inventory file...

**Review by Tekwriter1221 — COMMENTED** [review]
Only the missing word ("of") in line 66.

**Comment by ajacocks** [comment]
Thanks @Tekwriter1221 ! Change made.

**Review by 85matthew — APPROVED** [review]

**Pull request merged**